cisco-nexus-C3164PQ/config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Cisco Nexus C3164PQ",
+    "rules": [
+        {
+            "path": "version.chassis_id",
+            "regex": "Nexus3000 C3164PQ Chassis"
```

Comment:
Minor suggestion.
This more like to be `equals` or `contains` than `regex`. anyway the `regex` will still work.